multimodal/setup.py

```diff
@@ -28,35 +28,33 @@
     "tqdm",
     "boto3",
     "requests",
-    "jsonschema<=4.8.0",
-    "seqeval<=1.2.2",
-    "evaluate<=0.3.0",
-    "accelerate>=0.9,<0.16",
+    "jsonschema<4.18",
+    "seqeval<1.2.3",
+    "evaluate<0.4.0",
+    "accelerate>=0.9,<0.17",
     "timm<0.7.0",
     "torch>=1.9,<1.14",
     "torchvision<0.15.0",
     "fairscale>=0.4.5,<0.4.14",
     "scikit-image>=0.19.1,<0.20.0",
-    "smart_open>=5.2.1,<5.3.0",
     "pytorch_lightning>=1.8.0,<1.10.0",
-    "text-unidecode<=1.3",
+    "text-unidecode<1.4",
     "torchmetrics>=0.8.0,<0.9.0",
-    "transformers>=4.23.0,<4.26.0",
-    "nptyping>=1.4.4,<1.5.0",
-    "omegaconf>=2.1.1,<2.2.0",
+    "transformers>=4.23.0,<4.27.0",
+    "nptyping>=1.4.4,<2.5.0",
+    "omegaconf>=2.1.1,<2.3.0",
     "sentencepiece>=0.1.95,<0.2.0",
     f"autogluon.core[raytune]=={version}",
     f"autogluon.features=={version}",
     f"autogluon.common=={version}",
-    "pytorch-metric-learning>=1.3.0,<1.4.0",
+    "pytorch-metric-learning>=1.3.0,<2.0",
     "nlpaug>=1.1.10,<1.2.0",
     "nltk>=3.4.5,<4.0.0",
-    "openmim>0.1.5,<=0.2.1",
-    "defusedxml>=0.7.1,<=0.7.1",
-    "albumentations>=1.1.0,<=1.2.0",
-    "jinja2>=3.0.3,<3.1",
+    "openmim>0.1.5,<0.4.0",
+    "defusedxml>=0.7.1,<0.7.2",
+    "jinja2>=3.0.3,<3.2",
```

Comment:
jinja2 needs to be `<3.1` or it will error, at least it did in my PR before I dropped to `<3.1`.

Reply:
The error may have been from the `requirements_docs` dependencies though, so maybe `<3.2` is ok for the package depencenies, lets keep it `<3.2` and see if it passes

Reply:
Okay, let's see if the CI + document build can pass.